lib/logster/logger.rb

```diff
@@ -25,17 +24,21 @@ def override_level
       Thread.current[@logster_override_level_key]
     end
 
-    def chain(logger)
-      @chained << logger
+    def chained
+      @chained.keys
```

Comment:
Can this method be removed? (The presence of `@chained` and `chained` is also confusing)

Reply:
Oh I just noticed the [PR](https://github.com/discourse/discourse/pull/27663/files) is accessing `Logster.logger.instance_variable_get(:@chained).delete(Logster.logger.chained.first)`. Is there a better way around this? It feels weird that Discourse knows about internals.

Reply:
Actually, this change breaks backwards compatibility so I need to think of something else.

Reply:
Logster::Logger#chained is technically a public API